README.md

```diff
@@ -69,27 +70,20 @@ Please comment on the enhancement issue to:
 Please do not comment on the enhancement issue to:
 - discuss a detail of the design, code or docs. Use a linked-to-issue or PR for that
 
-## Enhancements Tracking Spreadsheet
+## Enhancements Tracking Board
```

Comment:
Changing this would break any links to this section of the Readme in and outside Kubernetes org repositories.

Can we keep the header and add a note that tracking sheets are deprecated right after the header?

Something like:

```
## Enhancements Tracking Board

<info about the boards>

 ## Enhancements Tracking Spreadsheet

<note the deprecation as a blockquote in bold and link to the archived page>
```

Reply:
Ughh let me fix this. One sec.

Reply:
I just removed deprecation entirely since I clearly explain which releases use what. Thanks for catching this.

https://github.com/kubernetes/enhancements/pull/3512

Reply:
Thanks! This fixes the concern.